### PR TITLE
extend list of requirements for PRs, add list of maintainers

### PR DESCRIPTION
**Pull request by boegel**
preview at http://boegel-eb.readthedocs.io/en/reqs_maintainers/Contributing.html and http://boegel-eb.readthedocs.io/en/reqs_maintainers/Maintainers.html

### CODE REVIEW

**Comment by migueldiascosta** [comment]
additional points to consider (don't affect the repositories but may be helpful anyway):

- guidelines for titles of PRs?
- usage of github labels?

otherwise, lgtm

**Comment by boegel** [comment]
@migueldiascosta The policy for PR titles should indeed be mentioned in the `Contributing` page as well, I'll port it from https://github.com/hpcugent/easybuild/wiki/Policy-for-easyconfig-pull-requests#use-a-standard-format-for-your-pr-title

I've been thinking about labelling issues and PRs, that's definitely going to be more important when we have multiple maintainers, but let's keep that a separate discussion, to be continued.

File: docs/Contributing.rst [hunk 1]
```diff
@@ -390,6 +390,30 @@ For example, to push the ``mybranch`` branch to your fork of the GitHub reposito
...
+
+Please use a descriptive (short) title for your pull requests, and clarify (if needed) in the pull request description.
+
+If any other pull requests are required, refer to them in the description using ``#<PR number>`` (only for pull requests
+to the same repository), or by copy-pasting the URL to the pull request.
+
+For pull requests to the easyconfig repository, we recommend using this format for the pull request title
+when contributing new easyconfig files::
+
+  {<moduleclass>}[<toolchain>] <software name> <software version> <extra info>``
```
Comment (migueldiascosta) [inline docs/Contributing.rst]: inconsistent backticks

File: docs/Maintainers.rst [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+--------
+
+EasyBuild maintainers should meet the following criteria:
+
+* being sufficiently familiar with the contribution procedure (see :ref:`contributing`)
+* frequently contributing to the relevant repositories
+* being an active member of the EasyBuild community for a significant amount of time, i.e.:
+
+  * following the EasyBuild mailing list and participating in discussions (see :ref:`getting_help`)
+  * actively participating in the EasyBuid IRC or Slack channel (see :ref:`getting_help`);
```
Comment (verdurin) [inline docs/Maintainers.rst]: EasyBuild

File: docs/Unit-tests.rst [hunk 3]
```diff
@@ -75,6 +75,7 @@ For each of the available easyconfig files, the following aspects are tested:
 * making sure that the specified sanity check paths adher to the requirements, i.e. only (and both) the ``files``/``dirs``
```
Comment (verdurin) [inline docs/Unit-tests.rst]: adhere

File: docs/index.rst [hunk 4]
```diff
@@ -114,7 +114,9 @@ Having trouble? We'd like to help!
 * :ref:`Search <search>` this documentation collection
 * Search for information in the `archives`_ of the easybuild@lists.ugent.be mailing list or `subscribe`_ to post a question.
 * Did you try ``eb --help``?
-* Ask a question in the `#easybuild IRC channel`_ on the Freenode network
+* Ask a question in the `#easybuild IRC channel`_ on the Freenode network,
+  or in the EasyBuild Slack channel https://easybuild.slack.com/
+  (subscribe via https://easybuild-slack.herokuapp.com)
 * Consider participating to an EasyBuild `conference call <https://github.com/hpcugent/easybuild/wiki/Conference-calls>`_
```
Comment (verdurin) [inline docs/index.rst]: participating on

File: docs/Contributing.rst [hunk 5]
```diff
@@ -606,14 +665,24 @@ does require a bit of practice because of the density of the provided informatio
 
 .. _contributing_review_process_test_reports:
 
-Test reports for easyconfig contributions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Test reports for easyconfig contributions (``upload-test-report``)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-For easyconfig contributions, an accompanying **test report must be submitted** to confirm that the touched easyconfig files (still) work as expected.
+For easyconfig contributions, one ore more accompanying **test reports must be submitted** to confirm that the added and/or changed easyconfig files (still) work as expected.
```
Comment (verdurin) [inline docs/Contributing.rst]: one or more

File: docs/Contributing.rst [hunk 6]
```diff
@@ -606,14 +665,24 @@ does require a bit of practice because of the density of the provided informatio
...
 
 We recommended that you submit a test report for your own easyconfig pull requests.
```
Comment (verdurin) [inline docs/Contributing.rst]: We recommend

**Comment by boegel** [comment]
@easybuilders/easybuild-docs-maintainers Good to go now imho.

**Pull request merged**